src/main/java/burp/Settings.java

```diff
@@ -8,7 +8,7 @@ public class Settings {
     private final String tlsFingerprintFilePath = "TlsFingerprintFilePath";
 
     public static final String DEFAULT_ADDRESS = "127.0.0.1:8887";
-    public static final int DEFAULT_TIMEOUT = 10;
+    public static final String DEFAULT_TIMEOUT = "10";
```

Comment:
This must be an `int` because the golang backend expects it to be.

Reply:
https://github.com/kmahyyg/burp-awesome-tls/blob/aea94ecfb975a7fa87ecb6280a41585bf2876f3c/src/main/java/burp/Settings.java#L58

When the getter is called by golang backend, it is parsed to int.